templates/helm/values.schema.json

```diff
@@ -203,6 +203,10 @@
         "pattern": "(^$|^.*=.*$)"
       }
     },
+    "deletionPolicy": {
+      "type": "string",
+      "enum": ["delete", "retain"]
```

Comment:
Since this enum is already enforced by the runtime [here](https://github.com/RedbackThomson/ack-runtime/blob/1e572bced2a58f806c54e04fe09c12eac307c6b0/apis/core/v1alpha1/deletion_policy.go#L25), what do you think about just making this a free-form string so that there is one fewer place where an enum needs to be updated if the values ever change?

Reply:
I want this here because I'd like the client-side Helm validation to fail if they try to add anything other than those values. Otherwise they would have to check the controller pod logs to understand what went wrong